worker/runc/runc.go

```diff
@@ -1,3 +1,6 @@
+//go:build linux
+// +build linux
```

Comment:
Can we just use the `_linux.go` suffix?

Reply:
Why is it better in this case where there are no other files with other suffixes?

Btw, this is so that `go vet ./...` does not try to build this package for windows.